src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -174,6 +174,16 @@ def get(self, uid):
         result = self.proxy('GET', 'user', {'uid': uid})
         return self._append_uid(result)
 
+    @Endpoint()
+    @ReadPermission
+    def getEmail(self):
+        emails = []
+        uids = eval(self.list())
```

Comment:
I'm not sure if eval is really required here. What is it used for and can't it be replaced by something else?

Reply:
The value returned by `self.list()` is a string containing multiple uids, I used this function to get an array of uids.